user-service/model/repository.js

```diff
@@ -13,6 +13,7 @@ mongoose.connect(mongoDB, { useNewUrlParser: true, useUnifiedTopology: true })
 
 let db = mongoose.connection
 db.on('error', console.error.bind(console, 'MongoDB connection error:'))
+db.collection('usermodels')

```

Comment:
Good idea! I like this cos it portrays the code intention clearer (i.e. this connection should only access this collection) - but just fyi I don't think the code actually needs this because mongoose operations are performed on models, which tells MongoDB the collection to operate on (e.g. `UserModel.findOne` will always be performed on the Users collection)

Reply:
I see, thanks for the heads up! Interesting to see that mongoose classes automatically maps it